### PR TITLE
Enable exclude-files

### PR DESCRIPTION
**Pull request by egallyamov**
Modifs to init.py and README

### CODE REVIEW

File: src/enacrestic/__init__.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 + ~/.enacrestic/bkp_include
   This is given to --files-from option
++ ~/.enacrestic/bkp_exclude
+  This is given to --exclulde-file option
```
Comment (sbancal) [inline src/enacrestic/__init__.py:14]: May I ask to fix the typo here ? `--exclulde-file`
thanks 😉

File: src/enacrestic/__init__.py [hunk 2]
```diff
@@ -45,6 +47,7 @@
 ENACRESTIC_PREF_FOLDER = os.path.expanduser('~/.enacrestic')
 RESTIC_USER_PREFS = {
     'FILESFROM': os.path.join(ENACRESTIC_PREF_FOLDER, 'bkp_include'),
+    'EXCLUDEFILES': os.path.join(ENACRESTIC_PREF_FOLDER, 'bkp_exclude'),
```
Comment (sbancal) [inline src/enacrestic/__init__.py:50]: As restic's parameter is `--exclude-file` (singular), could you rename this variable to `EXCLUDEFILE` please ?

File: src/enacrestic/__init__.py [hunk 3]
```diff
@@ -619,7 +623,8 @@ def _start_when_systray_available(self):
 
         self.tray_icon.setContextMenu(menu)
 
-        self.state.connect_to_gui(self.tray_icon, info_action, autostart_action)
+        self.state.connect_to_gui(
```
Comment (sbancal) [inline src/enacrestic/__init__.py:626]: As this doesn't change anything, could it be removed from PR ?

**Review by sbancal — CHANGES_REQUESTED** [review]
Thanks for your contribution.
I asked some little changes ... to match project's standard. Thanks in advance.

Please confirm that you also tested the changes with the case when the user has not prepared a `~/.enacrestic/bkp_exclude` file. This is important.

Regards,
Samuel

File: src/enacrestic/__init__.py (hunk 1 above)
Comment (egallyamov) [reply src/enacrestic/__init__.py]: Done

File: src/enacrestic/__init__.py (hunk 2 above)
Comment (egallyamov) [reply src/enacrestic/__init__.py]: Done

File: src/enacrestic/__init__.py (hunk 3 above)
Comment (egallyamov) [reply src/enacrestic/__init__.py]: Done

**Review by egallyamov — COMMENTED** [review]
Indeed I had to add couple of lines to make the code work when bkp_exclude is not provided

**Comment by sbancal** [comment]
Dear egallyamov,

Thank you for your contribution. I'm sorry that I didn't see your corrections earlier, otherwise I would have integrated it really faster.

Since Other functionnalities came to the source code and broke your pull request, I injected / adapted your contributions in a new commit.

The new release "0.1.7" is now available and take into account the `~/.enacrestic/bkp_exclude` as you suggested.
You can upgrade your ENACrestic installation following https://github.com/EPFL-ENAC/ENACrestic#upgrade documentation.

Here is the dedicated documentation chapter : https://github.com/EPFL-ENAC/ENACrestic#edit-enacresticbkp_exclude-file-optional

Best regards,
Samuel

**Pull request closed without merging**